integrationTesting/tests/organize/tags/remove.spec.ts

```diff
@@ -37,16 +37,24 @@ test.describe('Tags manager', () => {
       'delete'
     );
 
+    const tagToDelete = page.locator(`text="${ActivistTag.title}"`);
+    const deleteButton = page.locator('[data-testid=TagChip-deleteButton]');
+
     await page.goto(appUri + `/organize/1/people/${ClaraZetkin.id}`);
+    await tagToDelete.waitFor({ state: 'visible' });
+
+    await tagToDelete.hover();
+    await deleteButton.waitFor({ state: 'visible' });
 
-    await page.locator(`text="${ActivistTag.title}"`).hover();
     await Promise.all([
       page.waitForRequest((req) => req.method() == 'DELETE'),
-      page.locator('[data-testid=TagChip-deleteButton]').click(),
+      deleteButton.click(),
     ]);
 
     moxy.setZetkinApiMock(`/orgs/1/people/${ClaraZetkin.id}/tags`, 'get', []);
 
+    await tagToDelete.nth(2).waitFor({ state: 'hidden' });
```

Comment:
The `nth(2)` feels a bit risky. Is this because the title of the tag is "Active" and that string also occurs in other places on the page?

Reply:
This is now `await tagToDelete.waitFor({ state: 'hidden' });` thanks to changing the tag from `Activist` to `Plays guitar`.